README.md

```diff
@@ -94,9 +96,31 @@ def user_transaction_counts(transactions):
 
 If a Data Source input to the Feature View has `data_delay` set, then that delay will still be factored in to constructing training data sets but does not impact when the job can be triggered with the materialization API.
 
+Following is an example of definition of a `FeatureTable`
```

Comment:
probably not necessary imo since there is nothing special about this (vs FV which requires special args)

Reply:
I would prefer to leave an example in readme.

Reply:
I think it clutters it and doesn't add anything. If we document every aspect of Tecton here, it is more likely to fall out of date